pages/index.js

```diff
@@ -1,20 +1,23 @@
-import Head from 'next/head'
-import Sidebar from '../components/sidebar'
-import Header from "../components/Header"
-import Container from '../components/Container'
+import Head from "next/head";
```

Comment:
All files actually pass through `_app.js` and not `index.js` since we're using Next.js. Please move the ThemeProvider HOC to `_app.js` instead.